testdata/baselines/reference/submoduleAccepted/fourslash/inlayHints/inlayHintsJsDocParameterNames.baseline.diff

```diff
@@ -0,0 +1,36 @@
+--- old.inlayHintsJsDocParameterNames.baseline
++++ new.inlayHintsJsDocParameterNames.baseline
+@@= skipped -1, +1 lines =@@
+ y.foo(1, 2)
+       ^
+ {
+-  "label": "a:",
++  "label": [
++    {
++      "value": "a",
++      "location": {},
```

Comment:
Location is empty in baselines; are we missing some code that can baseline these?

Reply:
Hm, but there's only a string here, not something with a location? Maybe I'm missing something

Reply:
This label stands for the identifier `a`, so its value is `"a"`, and the location should point to the original parameter identifier, but location fields are being omitted when computing the diffs. They do show up in the `.baseline` file. If there's something I can do to make this all less confusing, let me know, I haven't put a ton of effort into the diff fixing code because we're eventually getting rid of it.

Reply:
I think the thing I was missing was that the Strada baseline says:

```json
{
  "text": "a:",
  "position": 88,
  "kind": "Parameter",
  "whitespaceAfter": true
}
```

Whereas the new baseline says:

```json
{
  "position": {
    "line": 6,
    "character": 6
  },
  "label": [
    {
      "value": "a",
      "location": {
        "uri": "file:///a.js",
        "range": {
          "start": {
            "line": 3,
            "character": 17
          },
          "end": {
            "line": 3,
            "character": 18
          }
        }
      }
    },
    {
      "value": ":"
    }
  ],
  "kind": 2,
  "paddingRight": true
}
```

So with that context, it makes sense what is happening here.

Reply:
Ah yes, that's because "interactive inlay hints" is always true in Corsa now, so we always return inlay hint label parts.